### ISSUE TITLE
KeyError: 'content-length' - This header is no longer being returned

### ISSUE THREAD

**Issue by sw564a** [cwtravis/bitbucket-asoc-sast-linux#1]
The script expects a content-length header in order to download the file, e.g., from url = "https://cloud.appscan.com/api/SCX/StaticAnalyzer/SAClientUtil?os=linux".  At the current time this header is not returned and the script fails with a message KeyError: "content-length.

Log messages

DEBUG: https://cloud.appscan.com:443 "GET /api/SCX/StaticAnalyzer/SAClientUtil?os=linux HTTP/1.1" 200 None
Traceback (most recent call last):
  File "/RunSAST.py", line 401, in <module>
    pipe.run()
  File "/RunSAST.py", line 103, in run
    appscanPath = self.getSAClient(saclientPath)
  File "/RunSAST.py", line 185, in getSAClient
    file_size = int(r.headers["content-length"])
  File "/usr/local/lib/python3.7/site-packages/requests/structures.py", line 52, in __getitem__
    return self._store[key.lower()][1]
KeyError: 'content-length'

**Comment by cwtravis** [comment]
Hello and sorry for the delayed response. Remember this in an open source plugin and not maintained by the AppScan development team.

AppScan on Cloud no longer returns the content-length header. This was breaking the percentage based download status messages for the SAClient.zip file. I changed the status messages to instead print the number of MB downloaded so far. This still conveys progress on the download even without the percentages. I tested this on my personal bitbucket account and my pipeline now works fine. Let me know if the fix works for you.

Commit df4a8ffc6b031224815f09a753157d0d456a10fe addresses this issue.
